src/types/quoting.ts

```diff
@@ -31,4 +33,12 @@ export interface Settings {
   queries: {
     staleTime: number
   }
+  rpcUrls: {
```

Comment:
`Record<SupportedChainName, string>` will be better, so if new chain is added, it will yell at you saying you missed RPC URL here. The same pattern should be used in all other places where chain name is involved.